raven-grails3-example/build.gradle

```diff
@@ -41,7 +41,10 @@ dependencyManagement {
 }
 
 dependencies {
-    compile "com.getsentry.raven:raven-logback:7.3.0" // Sentry's (Raven) only dependency
+    compile "com.getsentry.raven:raven-logback:7.8.1"
+    compile "ch.qos.logback:logback-core:1.1.3"
+    compile "ch.qos.logback:logback-classic:1.1.3"
```

Comment:
With newer logback Grails kept raising an exception... Not sure if we want to make an issue out of that or what.